common/sgx/verifier.c

```diff
@@ -213,11 +214,34 @@ oe_result_t oe_sgx_add_claim(
     return OE_OK;
 }
 
+static const char* _parse_tcb_status(const oe_tcb_level_status_t* tcb_status)
+{
+    if (tcb_status->fields.revoked == 1)
+        return "revoked";
```

Comment:
Use macros for the tcb status strings for this function and ` _parse_tcb_status()` in tcbinfo.c.

Reply:
Added macros and move two parsing functions to the same place.